dicoogle/src/main/java/pt/ua/dicoogle/server/SearchDicomResult.java

```diff
@@ -279,7 +279,7 @@ public DicomObject next() {
 
                 result.putString(Tag.Modality, VR.CS, (String) sR.get("Modality"));
 
-                result.putString(Tag.SeriesNumber, VR.IS, "" + (String) sR.get("SeriesNumber"));
+                result.putString(Tag.SeriesNumber, VR.IS, "" + sR.get("SeriesNumber"));
```

Comment:
We might as well add an extra safeguard against #378 regardless of what the underlying index provides.
```suggestion
                String seriesNumber = String.valueOf(sR.get("SeriesNumber"));
                if (seriesNumber.endsWith(".0")) {
                    seriesNumber = seriesNumber.substring(0, seriesNumber.length() - 2);
                }
                result.putString(Tag.SeriesNumber, VR.IS, seriesNumber);
```